src/main/resources/application.yml

```diff
@@ -0,0 +1,36 @@
+spring:
+  jpa:
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQL95Dialect
+        ddl-auto: none
+        show_sql: false
+  datasource:
+    primary:
```

Comment:
We can add type: com.zaxxer.hikari.HikariDataSource and as I understood it should be enough. No need to define any Hikary specific classes or beans. Only publish DataSources